ush/python/pygfs/task/atm_analysis.py

```diff
@@ -186,12 +190,11 @@ def finalize(self: Analysis) -> None:
             biaslist.append([src, dest])
 
         gprefix = f"{self.task_config.GPREFIX}"
-        gdate = to_YMDH(self.task_config.previous_cycle)
-        gsuffix = f"{gdate}" + ".txt"
+        gsuffix = f"{to_YMDH(self.task_config.previous_cycle)}" + ".txt"
         aprefix = f"{self.task_config.APREFIX}"
-        adate = to_YMDH(self.task_config.current_cycle)
-        asuffix = f"{adate}" + ".txt"
+        asuffix = f"{to_YMDH(self.task_config.current_cycle)}" + ".txt"
 
+        logger.info(f"Copying DATA/ {gprefix}*{gsuffix} to COM/ as {aprefix}*{assuffix}")
```

Comment:
Test of atm final job aborted in prufstam due to `assuffix` being undefined.   This is a typo.   Variable name is `asuffix`.   Remove extra `s` and rerun completed without error.   Expected files copied to ROTDIR.